server/portal/apps/onboarding/steps/system_access_v3.py

```diff
@@ -17,13 +18,28 @@ def push_system_credentials(user, public_key, private_key, system_id, skipCreden
     """
     logger.info(f"Adding user credential for {user.username} to Tapis system {system_id}")
     data = {'privateKey': private_key, 'publicKey': public_key}
-    user.tapis_oauth.client.systems.createUserCredential(
+    client = service_account()
+    client.systems.createUserCredential(
```

Comment:
Does this need to be from the service account?

Reply:
I believe it does if the system isn't shared publically. I don't know how safe that assumption is going to be with all the systems we're going to set up, so I think it makes sense to err on the side of using a service client if we have one.

Reply:
Hmm, I think public systems should be a safe assumption on our portals tenant where we have the service account. I'm leaning not using the service account unless absolutely necessary

Reply:
Sure, makes sense. I committed the change.